integration/dockerfiles/Dockerfile_test_dockerignore_relative

```diff
@@ -0,0 +1,5 @@
+# This dockerfile makes sure Dockerfile.dockerignore is working
```

Comment:
Can we remove this from integration test and rename this to "Dockerfile_dockeringore_relative"

Reply:
@tejal29 I'm not 100% sure how things work: if the file doesn't have the word `test` in it, then it won't be included in integration tests or is there more to do?

Reply:
ah you answered below, thx :)